src/ecs_cluster/ecs_client.py

```diff
@@ -187,9 +189,13 @@ def get_latest_task_definition_arn(self, cluster_name, service_name):
             status='ACTIVE',
             sort='DESC'
         )
-
-        latest_arn = response['taskDefinitionArns'][0]
-        return latest_arn
+        for task_definition_arn in response['taskDefinitionArns']:
```

Comment:
Is this the right logic? Iterating over the task definitions until you find one with the right tags?
I was thinking was only going to get the latest and then check if it has the tags or not.

Reply:
I thought we wanted to deregister the most recent image tagged with `ecs-cluster`